CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### 🔧 Internal changes
 
+## [2.11.2] - 2025-9-12
```

Comment:
Don't create a new release section, but instead add this entry to the unreleased section